v2/session/migration/from-auth0/session-migration.mdx

```diff
@@ -7,91 +7,6 @@ hide_title: true
 <!-- COPY DOCS -->
 <!-- ./session/migration/from-auth0/session-migration.mdx -->
 
-# Session Migration
+import Redirector from '/src/components/Redirector'
 
-In this section we will go over how to migrate Auth0 user sessions to SuperTokens sessions.
-
```

Comment:
there is another migration guide in community docs, make that redirect to `/docs` path